tests/unit/scripting.tcl

```diff
@@ -314,6 +319,12 @@ start_server {tags {"scripting"}} {
         catch {run_script {return redis.call()} 0} e
         set e
     } {*one argument*}
+   
+    test {EVAL - No arguments to server.call/pcall is considered an error} {
```

Comment:
Shall we add a sanity test for `pcall` as well?

Reply:
There is one now after I added the find/replace testing